commands/new_function.go

```diff
@@ -92,6 +92,10 @@ func runNewFunction(cmd *cobra.Command, args []string) error {
 
 	appendMode := false
 	if len(yamlFile) > 0 {
+		if (yamlFile == defaultYAML) && usingDefaultYaml {
```

Comment:
I think this can be simplified.   I wonder if `if (len(yamlFile) > 0)  && (usingDefaultYaml==false) {` would remove the need for the inner `if`?

Reply:
your right. if `usingDefaultYaml` is true it means always `yamlFile == defaultYAML` true. I will remove the nested check and add it as you suggested